databricks-sdk-java/src/main/java/com/databricks/sdk/core/retry/RequestBasedRetryStrategyPicker.java

```diff
@@ -0,0 +1,42 @@
+package com.databricks.sdk.core.retry;
+
+import com.databricks.sdk.core.DatabricksConfig;
+import com.databricks.sdk.core.http.Request;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class RequestBasedRetryStrategyPicker implements RetryStrategyPicker {
+  private static final List<Request> NON_IDEMPOTENT_REQUESTS =
```

Comment:
I thought we were going to do this the other way around: treat everything as non-idempotent but have an allow-list for the things which are allowed to retry even when we're not sure the server has processed the request.